### PR TITLE
Doris-spark connector examples

### PR DESCRIPTION
**Pull request by eyesmoons**
## Proposed changes
Provides some examples for doris spark connector using java and scala language.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)
- [ ] Code refactor (Modify the code structure, format the code, etc...)
    i. Two code files has been added.DorisSparkConnectionExampleJava.java and DorisSparkConnectionExampleScala.scala
- [ ] Optimization. Including functional usability improvements and performance improvements.
- [ ] Dependency. Such as changes related to third-party components.
- [ ] Other.

### CODE REVIEW

File: samples/doris-demo/spark-demo/org/apache/doris/java/DorisSparkConnectionExampleJava.java [hunk 1]
```diff
@@ -0,0 +1,142 @@
+package org.apache.doris.java;
```
Comment (hf200012) [inline samples/doris-demo/spark-demo/org/apache/doris/java/DorisSparkConnectionExampleJava.java:1]: add license head

File: samples/doris-demo/spark-demo/org/apache/doris/scala/DorisSparkConnectionExampleScala.scala [hunk 2]
```diff
@@ -0,0 +1,157 @@
+package org.apache.doris.scala
```
Comment (hf200012) [inline samples/doris-demo/spark-demo/org/apache/doris/scala/DorisSparkConnectionExampleScala.scala:1]: add license head

**Comment by hf200012** [comment]
Add a usage documentation

**Review by hf200012 — CHANGES_REQUESTED** [review]
Add a usage documentation

Add license to the java file

File: samples/doris-demo/spark-demo/org/apache/doris/scala/DorisSparkConnectionExampleScala.scala (hunk 2 above)
Comment (eyesmoons) [reply samples/doris-demo/spark-demo/org/apache/doris/scala/DorisSparkConnectionExampleScala.scala]: Usage documentation and license has been added.

**Review by hf200012 — APPROVED** [review]
LGTM

**Review by morningman — APPROVED** [review]

**Pull request merged**